db/column_family.cc

```diff
@@ -492,6 +492,22 @@ ColumnFamilyOptions ColumnFamilyData::GetLatestCFOptions() const {
   return BuildColumnFamilyOptions(initial_cf_options_, mutable_cf_options_);
 }
 
+uint64_t ColumnFamilyData::OldestLogReferenced() {
+  auto current_log = GetLogNumber();
+  auto imm_prep_log = imm()->GetMinLogContainingPrepSection();
```

Comment:
Similarly, I think it may be a better idea to still guard line 496 to 506 with if 2PC.